src/index.js

```diff
@@ -0,0 +1,42 @@
+/* eslint-disable indent */
+import './css/index.css';
+
+const todoObjectArray = [
+  {
+    id: 1,
+    title: 'Zoom meeting at 11 am',
+    completed: false,
+  },
+  {
+    id: 2,
+    title: 'Go to church...',
+    completed: false,
+  },
+  {
+    id: 3,
+    title: 'walking during the night',
+    completed: false,
+  },
+  {
+    id: 4,
+    title: 'write some codes',
+    completed: false,
+  },
+];
+
+const taskList = document.getElementById('task-list');
+
+function renderBook() {
+  taskList.innerHTML = '';
+  todoObjectArray.forEach((todoObject, index) => {
+    taskList.innerHTML += `<ul class="todoTask" id="${index + 1}">
+                  <li><input type="checkbox" id="checkbox" onchange="ChangeCompleted(${
+                    todoObject.id
+                  })" name="checkbox"></li>        
+                  <li class="todoName">${todoObject.title}</li>    
+                  <li><i class="fa fa-ellipsis-v" aria-hidden="true"></li>   
+                  </ul> `;
+  });
```

Comment:
- [ ] **Optional:** Kindly fix these to make your design cleaner.

![image](https://user-images.githubusercontent.com/27709832/176731008-2e81217d-9fba-4b4c-8727-280ae00eaf8f.png)